src/DynamoPackages/Package.cs

```diff
@@ -528,6 +528,7 @@ internal void MarkForUninstall(IPreferences prefs)
         {
             if (BuiltInPackage) 
             {
+                Analytics.TrackEvent(Actions.BuiltInPackageConflict, Categories.PackageManagerOperations, $"{Name } {versionName} marked to be unloaded");
```

Comment:
same question here - what indicates that this code path is a package conflict scenario?

Reply:
thanks for pointing these out @aparajit-pratap, my thought was that it would be good to note if any builtin packages were being uninstalled or unloaded at all - while it's not necessarily because of a conflict it seems likely, and grouping them all together under one analytics category seemed simpler, the extended description in analytics does call out exactly what the operation was.

We can get rid of these I guess.